CHANGELOG.md

```diff
@@ -20,6 +20,20 @@ New Features
 
   This will guarantee that your signaling traffic will terminate in Germany. For other possible values
   for region, please refer to this [table](https://www.twilio.com/docs/video/ip-address-whitelisting#signaling-communication).
+  If you specify an invalid value for `region`, `connect` will raise a [SignalingConnectionError](https://www.twilio.com/docs/api/errors/53000):
+
+  ```js
+  const { connect } = require('twilio-video');
+
+  try {
+    const room = await connect(token, {
+      region: 'foo'
+    });
+  } catch (error) {
+    assert.equal(error.code, 53000);
+    assert.equal(error.message, 'Signaling connection error');
+  }
+  ```
```

Comment:
Some small suggestions below:

By default, twilio-video.js connects to your nearest signaling server by automatically finding the signaling server connection with the lowest latency. ﻿Howe﻿ver﻿,﻿ in some cases,﻿ you may ﻿have a need to ﻿lock in to a specific region. Y﻿ou can﻿ now override this behavior by using a﻿ new ConnectOptions flag called region﻿.﻿ This will make sure that your signaling traffic will terminate in the region you specify. The default value for region is “gll“ (Global Low Latency). With this setting, twilio-video.js automatically finds the signalling server with the lowest latency. Please refer to this table for a complete regio﻿﻿ns list ﻿(JSDK-2338)

For example, to connect to “de1”:﻿
const { connect } = require(‘twilio-video‘);
const room = await connect(token, {
  region: ‘de1‘
});

Note, if you specify an invalid value for region, connect will raise a SignalingConnectionError:
const { connect } = require(‘twilio-video‘);

try {
  const room = await connect(token, {
    region: ‘foo‘
  });
} catch (error) {
  assert.equal(error.code, 53000);
  assert.equal(error.message, ‘Signaling connection error‘);
}